docs/usage/two-phase-transfer.md

```diff
@@ -5,50 +5,68 @@ protocol for distributed
 transactions](https://en.wikipedia.org/wiki/Two-phase_commit_protocol).
 
 Single-phase transfers post funds to accounts immediately when they
-are created. That is, they affect the `debits_posted` and
-`credits_posted` fields on relevant accounts.
+are created. That is, they affect the
+[`debits_posted`](../usage/accounts.md#debits_posted) and
+[`credits_posted`](../usage/accounts.md#credits_posted) fields on
+relevant accounts.
 
 In contrast to single-phase transfers, a two-phase transfer moves
 funds in stages:
 
 1. First, the pending transfer reserves funds. While reserved, they
-cannot be used by either the payer or payee. Only the `debits_pending`
-and `credits_pending` fields are modified on the relevant accounts at
-this point.
+cannot be used by either the payer or payee. Only the
+[`debits_pending`](../usage/accounts.md#debits_pending) and
+[`credits_pending`](../usage/accounts.md#credits_pending) fields are
+modified on the relevant accounts at this point.
 
-2. Later, the application creates another transfer — with flags either
-marked `post_pending_transfer` or `void_pending_transfer`. The former
-moves all (or part) of the reserved funds to the destination; the
-latter reverts them to the original account.
+2. Later, the application creates another transfer — with either of
+   the following flags:
+* [`post_pending_transfer`](../usage/transfers.md#flagspost_pending_transfer): Move all (or part) of the reserved funds to the pending transfer's destination.
+* [`void_pending_transfer`](../usage/transfers.md#flagsvoid_pending_transfer): Revert all of the reserved funds to the original account.
```

Comment:
Ok I know I suggested this change.. but I think we should expand this a bit (either here or in a follow-up paragraph).
Lets make it more clear that the money being moved/reverted is what is was originally added to the pending debits/credits, and that the pending transfer's modification to the pending debits/credits is undone.